uber/models.py

```diff
@@ -1150,7 +1150,7 @@ def _staffing_adjustments(self):
             old_staffing = self.orig_value_of('staffing')
             if self.staffing and not old_staffing or self.ribbon == c.VOLUNTEER_RIBBON and old_ribbon != c.VOLUNTEER_RIBBON:
                 self.staffing = True
-            elif old_staffing and not self.staffing or self.ribbon != c.VOLUNTEER_RIBBON and old_ribbon == c.VOLUNTEER_RIBBON:
+            elif old_staffing and not self.staffing or self.ribbon != c.DEPT_HEAD_RIBBON and self.ribbon != c.VOLUNTEER_RIBBON and old_ribbon == c.VOLUNTEER_RIBBON:
```

Comment:
This is actually a case where I'd use parentheses - even if it's not strictly necessary here, it's better to communicate how you want it to work.

That being said, I _think_ you do need it. If I understand correctly, Python will read this as:
"if the attendee used to be staffing, and they're not staffing anymore or they don't have a department head ribbon, and they don't have a volunteer ribbon, and they used to have a volunteer ribbon:"

Whereas you want it read as:
"if the attendee used to be staffing, and they're not staffing anymore or (they don't have a department head ribbon and they don't have a volunteer ribbon), and they used to have a volunteer ribbon:"